File: CS/CustomBarAndDockingController.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Linq;
using System.Text;
using DevExpress.XtraBars;
using DevExpress.XtraBars.Styles;

namespace WindowsFormsApplication242 {
    [System.ComponentModel.DesignerCategory("")]
    public class CustomBarAndDockingController : BarAndDockingController {
        public CustomBarAndDockingController(IContainer container)
            : base(container) {
        }
        public CustomBarAndDockingController() {
        }
        protected override void RegisterPaintStyles() {
            base.RegisterPaintStyles();
            AddSkinPaintStyle();
        }
        void AddSkinPaintStyle() {
            PaintStyles.Add(new CustomSkinBarManagerPaintStyle(PaintStyles));
        }
    }
}

File: VB/CustomBarAndDockingController.vb
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Linq
Imports System.Text
Imports DevExpress.XtraBars
Imports DevExpress.XtraBars.Styles

Namespace WindowsFormsApplication242
    <System.ComponentModel.DesignerCategory("")> _
    Public Class CustomBarAndDockingController
        Inherits BarAndDockingController

        Public Sub New(ByVal container As IContainer)
            MyBase.New(container)
        End Sub
        Public Sub New()
        End Sub
        Protected Overrides Sub RegisterPaintStyles()
            MyBase.RegisterPaintStyles()
            AddSkinPaintStyle()
        End Sub
        Private Sub AddSkinPaintStyle()
            PaintStyles.Add(New CustomSkinBarManagerPaintStyle(PaintStyles))
        End Sub
    End Class
End Namespace
